### ISSUE TITLE
Updates to G183 now no longer need underline on focus (or anytime!)

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#5162]
Based on a sufficient technique published by WCAG, I now have teams coming to me saying they don't need an underline on a link anytime except when it is hovered! This has come about as a result of changes in the always controversial [G183](https://www.w3.org/WAI/WCAG22/Techniques/general/G183). 

Originally, G183 was titled "Using a contrast ratio of 3:1 with surrounding text and providing **additional visual cues on focus** for links or controls where color alone is used to identify them" (my emphasis).

The test included the step:
> Check that moving keyboard focus to the link causes a visual enhancement (such as an underline, font change, etc.)

I want to make an observation that the original 'sufficient' technique was already pretty bad, given that it didn't specify an _underline_ be used. I believe they put themselves in a box by not having this technique strictly about links. They added "other controls", and so could not prescribe an underline. But at least it was listed first in the 'visual enhancements'. 

This preferece was entirely logical because if we weren't going to communicate that this was a link with the long-established default behaviour of links having persistent underlines, the very least we could do was show the underline when the link gets focus. 

But the present is worse than that. Due to changes implemented in https://github.com/w3c/wcag/pull/1553, the title and the tests for G183 are all now about hover(!?!)

The result is that I have teams creating links where unless one is lucky enough to have the ability to hover over them, there will _never_ be a way of figuring out whether a chunk of focused text is a link or a button until one activates it or is lucky enough to have an AT that will tell you.

<img width="130" height="143" alt="A set of three links each with a string 'Link text'. The second one has an underline. The third has a blue focus rectangle." src="https://github.com/user-attachments/assets/0f09f15a-8b34-40cd-8923-06656c100b1e" />

Figure 1: A link in three states: default, hovered and focused. An underline only appears on hover.

I guess the one silver lining in this experience is that at least the team is using an **underline** on hover. A reductive interpretation of this sufficient technique is that it is now sufficient to create a link which _never_ presents with an underline and _pass_ WCAG.

I get why we have difficulty creating failure techniques, which must end up with a consistent result of someone failing WCAG. But is there any reason to create horrible _sufficient_ techniques?

**I would like to propose either tightening this up and making a passing outcome that is unembarrassing, or jettisoning the whole thing. It is now counter productive.**

Some background at https://github.com/w3c/wcag/issues/2310 and https://github.com/w3c/wcag/issues/1272

**Comment by patrickhlauke** [comment]
The short answer is: correct, if the link text contrasts sufficiently with adjacent non-link text (>= `3:1` ratio) then no additional indication like an underline are needed.

And yes, this makes G183 weird in its current state.

**Comment by patrickhlauke** [comment]
adding underlines on hover/focus (or even in the non-focused/non-hovered state) is at best **advisory** if the link text already has sufficient contrast against non-link text.

beyond that, there's no further requirement that there need to be any more visual indicators/differentiators

**Comment by mraccess77** [comment]
I agree the keyboard bit implies that it's acceptable to have color alone or low contrast on links without underlines until focused.  Although I would consider a link without sufficient signifiers as sufficient contrast, underline, or something else even when not focused as a failure.
